qa/rpc-tests/wallet.py

```diff
@@ -371,5 +412,5 @@ def Test():
         "debug": ["net", "blk", "thin", "mempool", "req", "bench", "evict"],  # "lck"
         "blockprioritysize": 2000000  # we don't want any transactions rejected due to insufficient fees...
     }
-    # "--tmpdir=/ramdisk/test",
-    t.main(["--nocleanup", "--noshutdown"], bitcoinConf, None)
+    # "--tmpdir=/ramdisk/test", "--srcdir=../../debug/src"
+    t.main(["--srcdir=../../debug/src", "--tmpdir=/ramdisk/test", "--nocleanup", "--noshutdown"], bitcoinConf, None)
```

Comment:
is this something left behind or do you really need it?  why is it necessary to specify a tempdir or srcdir here?

Reply:
Thanks, will keep them as a comment only.